src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -40,8 +41,13 @@ const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row'
   },
+  containerSigned: {
+    backgroundColor: colors.babyBlue
+  },
+  containerAnon: {
+    backgroundColor: colors.halfBaked
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new styles `containerSigned` and `containerAnon` are introduced. Ensure that these styles are applied correctly based on the chat type.



<!-- This is an auto-generated comment by OSS CodeRabbit -->